.github/workflows/build.yml

```diff
@@ -55,6 +55,17 @@ jobs:
         run: dotnet build --configuration Release --nologo --verbosity minimal
       - name: Initialise RDMP
         run: dotnet run -c Release --project Tools/rdmp/rdmp.csproj -- install --createdatabasetimeout 180 "(localdb)\MSSQLLocalDB" TEST_ -e
+      - name: Populate Databases.yaml 
+        shell: bash
+        run: |
+          cat > ./Tools/rdmp/Databases.yaml << EOF
+          CatalogueConnectionString: Server=(localdb)\MSSQLLocalDB;Database=TEST_Catalogue;Trusted_Connection=True;TrustServerCertificate=true;
+          DataExportConnectionString: Server=(localdb)\MSSQLLocalDB;Database=TEST_DataExport;Trusted_Connection=True;TrustServerCertificate=true;
+          EOF
+      - name: Run integration test scripts
+        run: dotnet run -c Release --project Tools/rdmp/rdmp.csproj -- -f ./scripts/create_list_destroy_catalogue.yaml
```

Comment:
Nice. Maybe we could run this as a "test" so we get the extra coverage from it counted towards Coveralls etc?